pages/ksphere/kommander/1.1/tutorials/on-prem/index.md

```diff
@@ -61,10 +61,15 @@ all:
 
 Once you have both `cluster.yaml` file and `inventory.yaml` file, you need to add a provisioner for your provider. Follow these steps:
 
-1. Navigate to **Administration / Cloud Providers**.
-2. Select the **Add Provider** button.
-3. Select **On Premise**.
-4. Enter a name for your provider and insert the full contents of your private ssh key, then hit **Verify** and **Save**.
+Make sure you have selected a workspace - for example the "Default Workspace".
+
+- Navigate to **Administration / Infrastructure Providers**.
```

Comment:
The previous list style (1. 2. 3. 4.) seems better to me in the context of sequential steps and I don't think the empty lines are necessary.

Reply:
It did not fully work the way it was due to the embedded HTML. Note how the numbers jump around.

<img width="269" alt="Screenshot 2020-07-23 at 14 38 45" src="https://user-images.githubusercontent.com/592445/88287206-46312800-ccf2-11ea-93bc-35c5744f04d6.png">